mmdet3d/models/backbones/base_pointnet.py

```diff
@@ -1,25 +1,32 @@
 # Copyright (c) OpenMMLab. All rights reserved.
 import warnings
 from abc import ABCMeta
+from typing import Optional, Tuple
 
+import torch
 from mmengine.model import BaseModule
 
 
 class BasePointNet(BaseModule, metaclass=ABCMeta):
     """Base class for PointNet."""
 
-    def __init__(self, init_cfg=None, pretrained=None):
+    def __init__(self,
+                 init_cfg: Optional[dict] = None,
```

Comment:
init_cfg use `OptMultiConfig` or `OptConfig` in `mmdet3d.utils`

Reply:
Thank you for your suggestion, I will revise it right away.